### PR TITLE
Added MovedToGradle for OD-Vic test cases in spring-boot

### PR DESCRIPTION
**Pull request by hermya**
Added MovedToGradle for OD-Vic test cases in spring-boot

### CODE REVIEW

**Review by darko-marinov — CHANGES_REQUESTED** [review]
Need more evidence for the claims.

File: gr-data.csv [hunk 1]
```diff
@@ -569,6 +569,15 @@ https://github.com/spring-projects/spring-boot,daa3d457b71896a758995c264977bdd14
...
 https://github.com/spring-projects/spring-boot,daa3d457b71896a758995c264977bdd1414ee4d4,spring-boot-project/spring-boot-actuator-autoconfigure,org.springframework.boot.actuate.autoconfigure.metrics.test.MetricsIntegrationTests.requestMappingIsInstrumented,NIO,InspiredAFix,,https://github.com/spring-projects/spring-boot/issues/38604;https://github.com/spring-projects/spring-boot/commit/203cd542c05318ca293de92c1f3063d154556c09
 https://github.com/spring-projects/spring-boot,daa3d457b71896a758995c264977bdd1414ee4d4,spring-boot-project/spring-boot-actuator-autoconfigure,org.springframework.boot.actuate.autoconfigure.metrics.test.MetricsIntegrationTests.restTemplateIsInstrumented,NIO,InspiredAFix,,https://github.com/spring-projects/spring-boot/issues/38604;https://github.com/spring-projects/spring-boot/commit/203cd542c05318ca293de92c1f3063d154556c09
+https://github.com/spring-projects/spring-boot,daa3d457b71896a758995c264977bdd1414ee4d4,spring-boot-project/spring-boot,org.springframework.boot.context.properties.bind.CollectionBinderTests.bindToSetShouldNotAllowDuplicateValues,ID,,,
```
[the remark this answers is not in the document]
Comment (hermya) [reply gr-data.csv:572]: I am currently trying to run IdFlakies on the project. There are some problems with running it (gradle with iDFlakies)

[the remark this answers is not in the document]
Comment (darko-marinov) [reply gr-data.csv:572]: If you can't confirm that the test is `ID` in Gradle, then just mark it `MovedToGradle` in `pr-data.csv` but don't add it in `gr-data.csv`.

[the remark this answers is not in the document]
Comment (hermya) [reply gr-data.csv]: Okay. Currently I removed all changed data from gr-data.csv. Will update in future once verified

**Pull request merged**